mkdocs.yml

```diff
@@ -85,7 +85,7 @@ nav:
     - TeX, LaTeX, etc (texlab, external): page/lsp-latex.md
     - Verilog/SystemVerilog: page/lsp-verilog.md
     - VHDL: page/lsp-vhdl.md
-    - Vimscript: page/lsp-vim.md
+    - Vimscript: page/lsp-vimscript.md
```

Comment:
If you intend to change this, you need to change `lsp-clients.json` vim name too.

Reply:
Maybe it's a good idea test the documentation and see if it's working...
You can test it with `make local-webpage`

Reply:
my bad, I am having installing docker, as apt local mirror(brazil) seems to be down...

Reply:
it seems a lot of issues are happening:

```
Unchanged content check: (wrong-type-argument stringp nil)
lsp-bash.md changed on disk; really edit the buffer? (y, n, r or C-h) 
Cannot resolve conflict in batch mode
```

Reply:
you should just run on terminal `make local-webpage`, that will generate a lot of files and start a local webserver on localhost:8888 to test the changes

Reply:
what's with all those errors in make lint? oof!

Reply:
I really don't know what you did to happen these errors

Reply:
I can't find why it's failing to build documentation with `make local-webpage` command

Reply:
backtrace: https://gist.github.com/elxbarbosa/e56eb17a25ba8f3d37f03106925cbc64